src/BestBooks.js

```diff
@@ -26,20 +29,52 @@ class BestBooks extends React.Component {
     });
   }
 
+  makeBook = async (newBook) => {
+    let url = 'http://localhost:3001/books';
+    let bookResult = await axios.post(url, newBook);
+
+    // console.log(bookResult.data);
+    this.setState({
+      books: [...this.state.books, bookResult.data]
+    })
+  }
+
+  deleteBook = async (id) => {
+    let url = 'http://localhost:3001/books';
+    await axios.delete(`${url}/${id}`)
```

Comment:
- For this, is there a second argument to be passed in the axios.delete call. I think we have to pass the data as the second arg:

`await axios.delete('http://localhost:3001/books', { data: { book: etc } });`